docs/zkapps/tutorials/03-deploying-to-a-network.mdx

```diff
@@ -265,12 +265,18 @@ After the transaction is included in a block, your smart contract is deployed!
 You ran the `zk config` command to:
 
 - Create a deploy alias
-- Create fee payer key pair at `/Users/<username>/.cache/zkapp-cli/keys/03deploy.json`
-✔ Create zkApp key pair at `keys/berkeley.json`
-✔ Add deploy alias to config.json
```

Comment:
Just checking - we don't need to add this anymore?

Reply:
yep, I just divided up the tasks and the deploy alias is added with the zk deploy command, not the zk config command